runtime/devnet/src/config/extension.rs

```diff
@@ -0,0 +1,80 @@
+#[cfg(test)]
```

Comment:
This test should be in the chain extension crate. We can remove the use of `Runtime`. This was only a test for clarity but lets keep what we can keep.